compose/networking.md

```diff
@@ -42,14 +39,6 @@ When you run `docker compose up`, the following happens:
 3.  A container is created using `db`'s configuration. It joins the network
     `myapp_default` under the name `db`.
 
-> **In v2.1+, overlay networks are always `attachable`**
->
-> Starting in Compose file format 2.1, overlay networks are always created as
-> `attachable`, and this is not configurable. This means that standalone
-> containers can connect to overlay networks.
->
-> In Compose file format 3.x, you can optionally set the `attachable` property
-> to `false`.
```

Comment:
This is still true with the Compose specification, by default overlay networks are created as `attachable` by default, I think we should keep a sentence about this point (without mentioning compose file version of course 😉 )

Reply:
Ok I just saw the mention of that in the `Multi-host networking` section, ignore me 😁